database/migration/V20180110_2047__createAttendeeWebsites.sql

```diff
@@ -0,0 +1,12 @@
+CREATE TABLE `attendee_extra_info` (
```

Comment:
Since I'm thinking of making this release backwards incompatible (the db is empty anyways and this is the first release that will be used can we condense the migration files.

Reply:
Also now the filenames don't match, ideally leave the extrainfos the way it was in the original migration and then add your new tables to the bottom of the same file.